enterprise/app/tap/tap.tsx

```diff
@@ -403,33 +409,57 @@ export default class TapComponent extends React.Component<Props, State> {
     let first = this.getSortDirection() == "asc" ? a : b;
     let second = this.getSortDirection() == "asc" ? b : a;
 
-    let firstStats = this.state.stats.get(first?.target?.label);
-    let secondStats = this.state.stats.get(second?.target?.label);
+    let firstTarget = first.target;
+    let secondTarget = second.target;
+
+    let firstStats = this.state.stats.get(first?.target?.label || "");
+    let secondStats = this.state.stats.get(second?.target?.label || "");
+
+    if (!firstStats && !secondStats) {
+      return 0;
+    } else if (!firstStats) {
+      return 1;
+    } else if (!secondStats) {
+      return -1;
+    }
+
+    if (this.getSortMode() == "target") {
```

Comment:
I don't think this matters a ton, but putting this block (target behavior) before the nullish stats checks would make the behavior more like what it was before, right?

Reply:
Yup, done